com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/MyLanguageServerImpl.java

```diff
@@ -75,12 +75,14 @@
       TextDocumentService textService,
       WorkspaceService workspaceService,
       Provider<LanguageClient> clientProvider,
-      SettingsProvider settingsProvider) {
+      SettingsProvider settingsProvider,
+      MyTextDocumentService customTextService) {
```

Comment:
Why do we need this reference?

Reply:
that is TextDocumentService which is the interface provided by lsp4j and the parameter that I need to set is workspaceFolder which I added in the MyTextDocumentService, in order to get this into the didOpen/didClose method